unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEditorHUDModel.cs

```diff
@@ -13,12 +13,25 @@ public record BackpackEditorHUDModel
         public Color eyesColor;
         public HashSet<string> forceRender = new ();
 
-        public AvatarModel ToAvatarModel()
+        public AvatarModel ToAvatarModel(Dictionary<string, string> extendedWearableUrns = null)
         {
+            List<string> wearables;
+
+            if (extendedWearableUrns != null)
+            {
+                wearables = new List<string>();
```

Comment:
Is this method called only once? We don't need to cache this List (or use Unity pool for it), right?